markdown/manual.md

```diff
@@ -94,127 +94,168 @@ Annual cost of living increases will be combined with any merit increases descri
+Employees are able to work from home and/or remotely and the organization will make all efforts to ensure a comparable work experience regardless of where an employee is located.
 
-We track time using Harvest. Hours worked must be recorded in Harvest by the end of each week (Sunday evening).
+Any time that you won't be available during the regular work day (9 PM to 5 PM locally) should be put on your calendar the organization's calendar.
 
 ### Vacation & Leave Policy
-We are closed on all Federal holidays and during the week between Christmas and the New Year.  We also have a few additional days off, please see [the attached holidays list](holidays.md).
+We are closed on all [Federal holidays](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/holidays-work-schedules-and-pay), the day after Thanksgiving, and during the week between Christmas and the New Year.  We also have a few additional days off, please see [the attached holidays list](holidays.md).
 
-We expect everyone to be able to get their work done in a timely fashion, and to not have to work on personal time.  Rested and refreshed people are better team members, and we strongly encourage enjoying life outside of the office.  Employees should take a *minimum* of two weeks of paid vacation per year, totaling 10 business days.
+We expect everyone to be able to get their work done in a timely fashion, and to not have to work on personal time.  Rested and refreshed people are better team members, and we strongly encourage enjoying life outside of the office.  
 
-To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates and location of your travel must be put onto the team calendar, and the team should be notified via email.  To track time off, please track the vacation time in Harvest.
+*Employees should take a minimum of two weeks of paid vacation per year, totaling 10 business days.*
 
-For personal, medical, or family leave, each employee is encouraged to take the current Family and Medical Leave Act maximum.
+For personal, medical, or family leave, each employee is encouraged to take the current [Family and Medical Leave Act](https://www.dol.gov/general/topic/benefits-leave/fmla) maximum.
 
 For maternity and paternity leave, employees are encouraged to take up to 16 weeks of leave.
 
+#### Taking Leave
 Employees must inform your supervisor in writing as far in advance as is reasonable for staff and project planning purposes.  All extended periods of leave must be discussed with the Executive Director.
 
+To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates your travel must be put onto the team calendar, and the team should be notified via email.
```

Comment:
In the past I made the mistake of thinking putting time off on the calendar under my seth [at] ogf email address constituted the "team calendar" since it was visible to the OGF team. Might be good to make it more explicit that it's a very specific calendar.